docs/docs-site/content/user/concept/_index.md

```diff
@@ -110,15 +110,44 @@ Clicking on the link will open-up the SQL selection:
 
 ![Click Gist](https://cdn.gethue.com/uploads/2020/03/editor_gist_open_presentation_mode.png)
 
-### Share to Slack
+### Import / Export
+
+Via the Home page, saved documents can be exported for backups or transferring to another Hue.
+
+## Slack
 Currently in **Beta**
 
-This integration increases the collaboration with others via Slack. It expands the ability to share query links or gists to the desired Slack channels which then unfurls in a rich preview for other members to view.
+This integration increases the collaboration with other users via Slack.
 
+### Share to Slack
+This feature expands the ability to share query links or gists to the desired Slack channels which then unfurls in a rich preview for other users. It even gives the user result file for the query if it has not expired.
 
-### Import / Export
+To manually set-up the Hue App, follow the steps mentioned in the [link here](https://docs.gethue.com/administrator/configuration/server/#manual-slack-app-installation)
```

Comment:
No need to hardcode `https://docs.gethue.com`, actually better to avoid it if we change the address later

Reply:
Also `link here` --> The [bot setup] is described in the admin section.